src/Employer/Employer.Web/Views/Part1/EmployerName/EmployerName.cshtml

```diff
@@ -50,7 +49,41 @@
                                     <label asp-for="NewTradingName" class="govuk-label govuk-!-font-weight-bold" for="NewTradingName">Add a new trading name</label>
                                     <span class="govuk-hint">This will also change the trading name on all your vacancies that are in draft or rejected.</span>
                                 </div>
-                                <input asp-for="NewTradingName" class="govuk-input govuk-input--width-20" id="NewTradingName" name="NewTradingName" type="text" >
+                                <input asp-for="NewTradingName" class="govuk-input govuk-input--width-20" id="NewTradingName" name="NewTradingName" type="text">
+                            </div>
+                        </div>
+                        <div class="govuk-radios__divider">or</div>
+                        <div class="govuk-radios__item">
+                            <input asp-for="SelectedEmployerNameOption" value="Anonymous" data-aria-controls="anonymous-conditional" id="anonymous" class="govuk-radios__input" type="radio" >
+                            <label class="govuk-label govuk-radios__label" for="anonymous">
+                                do not display a name (be anonymous)
+                                <span class="govuk-hint">This could mean fewer people see your vacancy</span>
+                            </label>
+                        </div>
+                        <div class="govuk-radios__conditional" id="anonymous-conditional">
+                            <div class="govuk-form-group">
+                                <fieldset class="govuk-fieldset">
+                                    <div class="govuk-form-group">
+                                        <label class="govuk-label" asp-for="AnonymousName">
+                                            Brief description of what the employer does
+                                        </label>
+                                        <span class="govuk-hint">For example, car manufacturer or clothing retailer.</span>
+                                        <span esfa-validation-message-for="AnonymousName" class="govuk-error-message"></span>
+                                        <input asp-for="AnonymousName" data-val-length-max="200" class="govuk-input govuk-input--width-20" type="text">
```

Comment:
As there is a character limit and error message associated with max limit, would it be better for this field to be a text area as it is also asking for a description so a sentence.

Also there is no min limit meaning somebody could submit a single character such as `'`.

Reply:
This is as per the story. I'll chat to Jithesh about minlength.